pkg/imgpkg/internal/util/level_logger.go

```diff
@@ -10,18 +10,19 @@ type LogLevel int
 
 // UIWithLevels wraps a ui.UI with logging levels
 type UIWithLevels interface {
-	goui.UI
-
 	Errorf(msg string, args ...interface{})
 	Warnf(msg string, args ...interface{})
 	Debugf(msg string, args ...interface{})
 	Tracef(msg string, args ...interface{})
 }
 
 const (
+	// LogTrace most verbose level
```

Comment:
seems a bit suspicious to put util into internal. dont we have classes that want to use logger (and if they do we really want ppl to impl their own for the interface)?

Reply:
I did do a full swipe and moved all our utils into the internal packages as they are internal things that the users might not care, but I can pull the logger out

Reply:
ok, sounds good, separate PR?

Reply:
I did it in a separate commit in this PR, but if you want I can separate this change from this PR